wordpress-mobile/WordPress-iOS
# Comment and Post likes: get relevant IDs

Ref: #15662
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/392

To test:

This updates WPKit to use the changes to obtain `siteID` and `postID`/`commentID` for likes. The IDs aren't used in WP yet, so just some smoke testing should do.
- Go to Notifications > Likes.
- Select a comment like and a post like.
- Verify the likes list appears as expected.


## Regression Notes
1. Potential unintended areas of impact
N/A. Feature incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature incomplete.

3. What automated tests I added (or what prevented me from doing so)
Added relevant WPKit tests.

PR submission checklist:
